dvc/repo/experiments/queue/base.py

```diff
@@ -131,15 +131,27 @@ def put(self, *args, **kwargs) -> QueueEntry:
     def get(self) -> QueueGetResult:
         """Pop and return the first item in the queue."""
 
-    def remove(self, revs: Collection[str]) -> List[str]:
+    def remove(
+        self,
+        revs: Collection[str],
+        _all: bool = False,
```

Comment:
```suggestion
        all_: bool = False,
```

When trying to avoid collisions with reserved keywords either use a trailing underscore or change the name to something else like `remove_all`.

Try not to use a leading underscore for this (like `_all`), since the leading underscore is usually used to indicate either that a var is supposed to be private or that it is intentionally unused.